prinz/src/test/scala/pl/touk/nussknacker/prinz/util/collection/immutable/MapMultisetTest.scala

```diff
@@ -0,0 +1,96 @@
+package pl.touk.nussknacker.prinz.util.collection.immutable
+
+import pl.touk.nussknacker.prinz.UnitTest
+
+import scala.collection.mutable.ListBuffer
+
+class MapMultisetTest extends UnitTest {
```

Comment:
think of tests for immutability - after change new instance is returned but what happens with the old one?